src/components/modules/banners/contributions/ContributionsBannerMobile.tsx

```diff
@@ -67,8 +67,8 @@ export const ContributionsBannerMobile: React.FC<ContributionsBannerMobileProps>
             <div css={styles.ctaContainer}>
                 <ContributionsBannerCta
                     onContributeClick={onContributeClick}
-                    ctaText={content.ctaText}
-                    ctaUrl={content.ctaUrl}
+                    ctaText={content.primaryCta?.ctaText || ''}
+                    ctaUrl={content.primaryCta?.ctaUrl || ''}
```

Comment:
Perhaps we just don't want to render this if `primaryCta` is undefinded?

Reply:
yes -
`{content.primaryCta && (<div css={styles.ctaContainer}>...`
Also we can support the 2nd cta here